neo/Network/RPC/NeoAPI.cs

```diff
@@ -0,0 +1,152 @@
+using Neo.Network.P2P.Payloads;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using Neo.Wallets;
+using System;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+
+namespace Neo.Network.RPC
+{
+    /// <summary>
+    /// NEO APIs throught RPC
+    /// </summary>
+    public class NeoAPI
```

Comment:
Why not use these classes directly?

```c#
public RpcClient RpcClient { get; }
public ContractClient ContractClient { get; }
public Nep5API Nep5API { get; }
public PolicyAPI PolicyAPI { get; }
```

Reply:
Hi Erik, 
I want to support a centralized class NeoAPI, so users can access all the functions in the RPC client module after initialize NeoAPI.

Reply:
This name is not easy to understand.

Reply:
Yes, it is confusing when the whole project can be called NEO API. How about ```NeoRemoteAPI```?

Reply:
`NeoVirtualClientTools`? aheuhauea

Reply:
Hi @vncoelho 
```RpcClientTools``` seems suitable for this class, and I will change the name.

These services are provided by Neo.dll, and the provider node needs to open ```RpcServer``` to support these services. Do you mean we need to add config for which functions will be enabled by ```RpcServer```?